docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md

```diff
@@ -42,9 +43,9 @@ clusterawsadm alpha bootstrap create-stack \
 These will be added to the control plane and node roles respectively when they are created.
 
 **Note:** If you used `clusterawsadm` 0.5.4 or earlier to create IAM objects for the Cluster API
-Provider for AWS, using `clusterawsadm` 0.5.5 or later will, by default, remove the bootstrap 
+Provider for AWS, using `clusterawsadm` 0.5.5 or later will, by default, remove the bootstrap
 user and group. Anything using those credentials to authenticate will start experiencing
-authentication failures. If you rely on the bootstrap user and group credentials, specify 
+authentication failures. If you rely on the bootstrap user and group credentials, specify
```

Comment:
This is my favorite part

Reply:
Only if you move from the alpha commands to the GA commands. The alpha command behaviour was preserved...

Reply:
I meant the whitespace removal 😸

Reply:
In any case, I've clarified the language.